packages/tuono-router/package.json

```diff
@@ -48,8 +48,7 @@
   },
   "dependencies": {
     "react-intersection-observer": "^9.13.0",
-    "vite": "^5.2.11",
-    "zustand": "4.4.7"
+    "vite": "^5.2.11"
```

Comment:
I think this can become a `devDependencies`

Reply:
Yes, also set it to a fixed version range: right now we have `5.4.11` installed